docs/driver-development.md

```diff
@@ -58,19 +58,19 @@ In your PR, edit the following files in the Universal Resolver root directory:
   * driver version (e.g. `0.1`)
   * DID spec version that the driver conforms to, with link to DID spec
   * DID method spec version (e.g. `0.1`), with link to DID method spec (or mark "missing")
-  * Docker image name (e.g. `exampleorg/uni-resolver-driver-did-example`) with link to Docker image at DockerHub
+  * Docker image name (e.g. `exampleorg/uni-resolver-driver-did-example`) with link to Docker image at docker-registry
```

Comment:
Hmm will people understand here what is meant by "docker-registry" ?

Reply:
What about `with link to hosted Docker image`